apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3635,7 +3635,7 @@ int tc_filesystem_main(int argc, char *argv[])
 #ifdef CONFIG_ITC_FS
 	itc_fs_main();
 #endif
-#if defined(CONFIG_TC_FS_PROCFS) && !defined(CONFIG_SMARTFS_MULTI_ROOT_DIRS)
+#if defined(CONFIG_TC_FS_PROCFS) && defined(CONFIG_FS_SMARTFS) && !defined(CONFIG_SMARTFS_MULTI_ROOT_DIRS)
```

Comment:
Could you please check below comments?
1. tc_fs_procfs.c is only for testing procfs, so tc_fs_smartfs_mksmartfs should be moved to somewhere.
2. Whatever CONFIG_SMARTFS_MULTI_ROOT_DIRS set or not, mksmartfs can be called.
    so, this config should be removed and tc must handle this.
```
void tc_fs_smartfs_mksmartfs(void)
{
	int ret;
#ifdef CONFIG_SMARTFS_MULTI_ROOT_DIRS
	ret = mksmartfs(SMARTFS_DEV_PATH, 1, false);
#else
        ret = mksmartfs(SMARTFS_DEV_PATH, false);
#endif
       ....
	TC_SUCCESS_RESULT();
}

Reply:
Ok. I got it. When i verify by just enabling CONFIG_SMARTFS_MULTI_ROOT_DIRS,

Reply:
you can check errno first, fs using errno for most error case. and fdbg also need to check